public/app/features/dashboard-scene/scene/layout-responsive-grid/ResponsiveGridLayoutManager.tsx

```diff
@@ -28,10 +29,17 @@ export class ResponsiveGridLayoutManager
 
   public readonly descriptor = ResponsiveGridLayoutManager.descriptor;
 
+  public constructor(state: ResponsiveGridLayoutManagerState) {
+    super(state);
+
+    //@ts-ignore
+    this.state.layout.getDragClassCancel = () => 'drag-cancel';
```

Comment:
Don't understand this ignore. `SceneCSSGridLayout` implements `SceneLayout` so we should be able to do that. Maybe its because `SceneCSSGridLayout` does not have the method in it's signature, and the `SceneLayout` interface is just ignored?

Anyways, if that's not fixed in scenes, then i think we should add `@ts-expect-error` with a comment why :)

Reply:
@dprokop you can't override a prototype function, this is just a temp fix as the drag and drop PR from kay will move the CSSGridLayout implementation into core grafana (or a copied impl), and that copy implements this function.

Reply:
and after that PR is merged we can remove this